packages/back-end/src/controllers/organizations.ts

```diff
@@ -463,6 +463,7 @@ export async function getOrganization(req: AuthRequest, res: Response) {
   req.checkPermissions("organizationSettings");
 
   const {
+    id,
```

Comment:
I added this in, so that I can pass the organization's Id in to the Stripe checkout session endpoint as the client_reference_id. That id is then persisted when Stripe hits the webhook for a successful checkout session, and we're using the client_reference_id in order to update the organization with it's stripeCustomerId, and later, the subscription info.

I went this route as I saw that a user's email address can be in multiple orgs, so this felt like the safest route. But, if id was omitted intentionally for reason's I'm not aware of, let me know and I can always try to accomplish this a different way.